api/services/S3SiteRemover.js

```diff
@@ -1,7 +1,18 @@
 const S3Helper = require('./S3Helper');
 const CloudFoundryAPIClient = require('../utils/cfApiClient');
 const config = require('../../config');
-const { logger } = require('../../winston');
+
+const handleError = (err) => {
```

Comment:
I'm a little confused as to the logic here. It looks like this does nothing if JSON parsing succeeds AND the message includes `Not found`, is that the intention? so it doesn't halt the promise chain? I'm not sure this is the best place for this logic OR maybe we can be more explicit about what this function is doing.

Reply:
that is correct ... we want to still be able to delete the site and continue the promise chain if the service does not exist

Reply:
gotcha. Yeah I think a comment would be helpful here. Does this logic apply to all the uses of this function?

Reply:
yes